### DISCUSSION TITLE
ImportError: cannot import name 'orthonormalize_rotation_shear' from 'habitat_sim.utils.common'

### DISCUSSION THREAD

**Discussion by Draculair** [facebookresearch/habitat-lab#1573] [Q&A] [answered]
I'm using ubuntu22.04 and installed the habitat following the instruction in the readme, but when I ran the "python examples/example.py", I got this error. The package version is habitat-lab 2.5, habitat-sim 2.1, habitat-baselines 2.5. Anyone helps? Thanks a lot

**Comment by mathfac** [comment]
Hi
Draculaaair,

It is expected that Habitat Lab and Sim versions should match. Then it’s
compatible.

**Reply by Draculair** [reply]
Thanks for response, but I just followed the instructions to install all packages in README, maybe you guys can help to update the README? And another question is that how can I install, should I download an older version of the codes and then pip install -e?

**Reply by Draculair** [reply, marked as answer]
sorry, I found the problem that I'm using python3.8.

**Comment by mathfac** [comment]
Thank you for letting us know about conda packages inconsistency. (cc @aclegg3)
